go.mod

```diff
@@ -187,3 +187,5 @@ replace github.com/gogo/protobuf => github.com/gogo/protobuf v1.3.2
 go 1.19
 
 retract v0.0.2
+
+replace github.com/btcsuite/btcwallet => github.com/roasbeef/btcwallet v0.11.1-0.20230426202031-f47d2877b6e0
```

Comment:
Given that latest `btcwallet` version is `0.16.8` isn't this quite a big regression?

Reply:
@GeorgeTsagk  - this is just a temporary replace that will be converted to a version bump (> 0.16.8) after these PRs are merged: https://github.com/btcsuite/btcwallet/pull/859, https://github.com/btcsuite/btcwallet/pull/860

Reply:
Yeah it's just the "psuedo" version.

Reply:
Gone now in the latest push.